sources/packages/backend/apps/test-db-seeding/src/test-seed-helper-services/institution.helper.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Institution } from "@sims/sims-db";
+import { Repository } from "typeorm";
+
+@Injectable()
+export class InstitutionHelperService {
```

Comment:
I do not see a reason to call these services with the "helper". They can be services in the same way as the other application have them.